CK3toEU4/Data_Files/configurables/religions/current_icons.txt

```diff
@@ -0,0 +1,3 @@
+# We have 153 icons in the religion strip.
```

Comment:
Wrong place for this file, It should be in dynamic subfolder.

Reply:
Fixed